hack/godep-save.sh

```diff
@@ -34,34 +36,31 @@ REQUIRED_BINS=(
 )
 
 pushd "${KUBE_ROOT}" > /dev/null
+  # sanity check that staging directories do not exist in GOPATH
+  error=0
+  for repo in $(ls ${KUBE_ROOT}/staging/src/k8s.io); do
+    if [ -e "${GOPATH}/src/k8s.io/${repo}" ]; then
+      echo "k8s.io/${repo} exists in GOPATH. Remove before running godep-save.sh." 1>&2
+      error=1
+    fi
+  done
+  if [ "${error}" = "1" ]; then
+    exit 1
+  fi
+
+  echo "Deleting vendor/ and Godeps/"
+  rm -rf "${KUBE_ROOT}/vendor" "${KUBE_ROOT}/Godeps"
```

Comment:
This makes me uncomfortable. I don't see any other hack/ scripts that directly rm -rf anything outside of a temp directory... Perhaps we could warn/error like above, rather than actually rm the directories ourselves?